x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/utils.tsx

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+export const getConnectedBadgeAriaLabel = (connectedCount: number) =>
+  i18n.translate('xpack.enterpriseSearch.connectorStats.connectedBadgeAriaLabel', {
+    defaultMessage: '{number} Connected connectors',
+    values: {
+      number: connectedCount,
+    },
+  });
+export const getConnectedConnectorsBadgeLabel = (connectedCount: number) => (
+  <FormattedMessage
+    id="xpack.enterpriseSearch.connectorStats.connectedBadgeLabel"
+    defaultMessage="{number} Connected"
+    values={{
+      number: connectedCount,
+    }}
+  />
+);
+export const getConnectedConnectorsTooltipContent = (
+  connectedCount: number,
+  isCrawler: boolean
+) => (
+  <EuiText size="xs">
+    {!isCrawler ? (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.connectorStats.connectedTooltip"
+        defaultMessage="{connectedCount} {completeConnectorsText} - Number of connectors which are complete in the last 30 minutes"
```

Comment:
```suggestion
        defaultMessage="{connectedCount} {completeConnectorsText} - Number of connectors that are configured and connected."
```

Is this right?

Or did you mean "Number of _syncs_ which completed in the last 30 minutes."?

Reply:
If there is no activity last 30 minutes, it will not considered connected. 

So it is the number of connectors which had "connected" in last 30 mins, some activity has to be there.

Reply:
@efegurkan so maybe "Number of connectors successfully configured in the last 30 minutes" ?

Reply:
connected 🤝 ?

Reply:
🤝